frontend/src/components/inputs/EntityForm.vue

```diff
@@ -5,19 +5,24 @@
                  :required="true"/>
       <TextInput label="Slug" v-model="editEntity.slug" :args="{ id: 'slug', maxlength: 128 }"
                  :required="true"/>
-      <h3 class="mt-3">Attributes</h3>
-      <div v-if="attributes" class="border-bottom border-top border-light my-2">
-        <div class="fw-bold my-2">Values to copy:</div>
-        <div v-for="attr in schema?.attributes || []" :key="`${attr.name}-${attr.id}`" class="mb-1">
-          <label :class="attributes[attr.name] ? 'cursor-pointer' : ''">
-            {{ attr.name }}
-            <sup v-if="requiredAttrs.includes(attr.name)" class="text-danger me-1"
-                 data-bs-toggle="tooltip" title="This value is required">*</sup>
-            <input type="checkbox" @change="onAttributeCopyChange($event.target.checked, attr.name)"
-                   :checked="editEntity[attr.name]"
-                   :disabled="!attributes[attr.name]"
-                   :class="attributes[attr.name] ? 'cursor-pointer' : ''" />
-          </label>
+      <div class="d-flex justify-content-between align-items-end">
+        <h3 class="mt-3">Attributes</h3>
```

Comment:
It looks better than mine, thanks!

I will propose a small suggestion - if the attributes are too many, it looks like:
![image](https://github.com/SUSE/aimaas/assets/27234545/ea159ac9-77b2-4b90-96d2-913f3c626466)

I don't insist on this one, but I think it will look better if we do:

```suggestion
        <h3 class="align-self-start">Attributes</h3>
```

Reply:
Yes. But also like the spacing above the "Attributes" heading. But that can be moved somewhere else. Let me push another commit...